core/src/main/java/io/github/fourlastor/game/demo/turns/Move.java

```diff
@@ -10,10 +10,12 @@
 import dagger.assisted.Assisted;
 import dagger.assisted.AssistedFactory;
 import dagger.assisted.AssistedInject;
+import io.github.fourlastor.game.demo.actions.AttackMelee;
 import io.github.fourlastor.game.demo.state.GameState;
 import io.github.fourlastor.game.demo.state.tiles.Tile;
 import io.github.fourlastor.game.demo.state.unit.Unit;
 
+// sheerst: should this be named PickMove?
```

Comment:
yep, I split it in https://github.com/fourlastor-jams/into-the-wip/pull/11/commits/3142fa354bdd570e1f0dc2a359df2ef245ee9eef and it makes more sense (it also removes the bug where you could quickly click two tiles)

I think `AttackMelee` should be a `TurnState` on its own, instead of being a scene2d `Action`. Similar to how `Move` in that commit is a `TurnState`